schema/src/main/scala/org/constellation/schema/serialization/SchemaKryoRegistrar.scala

```diff
@@ -113,6 +114,12 @@ object SchemaKryoRegistrar
         (classOf[HeightRange], 204, DefaultSerializer),
         (classOf[CheckpointBlockPayload], 205, DefaultSerializer),
         (classOf[FinishedCheckpointBlock], 206, DefaultSerializer),
+        (NodeMemberOfActivePool.getClass, 207, CompatibleSerializer),
+        (NodeNotMemberOfActivePool.getClass, 208, CompatibleSerializer),
+        (NodeJoinsTheCluster.getClass, 209, CompatibleSerializer),
+        (NodeLeavesTheCluster.getClass, 210, CompatibleSerializer),
+        (NodeKickedOutByHealthcheck.getClass, 211, CompatibleSerializer),
+        (classOf[NextActiveNodes], 212, CompatibleSerializer),
```

Comment:
why not `classOf[]` as it was used before?

Reply:
Can you elaborate?

Reply:
I mean, you used `Foo.getClass` instead of `classOf[Foo]` as it was used almost everywhere else. that's why I'm asking ;)

Reply:
Ok you mean e.g. the line 121. That's how we got the class for the case objects. classOf[] doesn't work for case objects AFAIR

Reply:
yup. Ok, didn't know